src/ResourceManager/Storage/Commands.Management.Storage/StorageAccount/StorageManagementClient.cs

```diff
@@ -17,7 +17,7 @@
 using Microsoft.Azure.Management.Storage;
 using System;
 
-namespace Microsoft.Azure.Commands.Compute
+namespace Microsoft.Azure.Commands.Management.Storage
```

Comment:
Yes, saw that. But what do you think of this namespace? It seems to me it's a bit confusing to put ```Commands``` and ```Management``` together, whereas the latter one usually means management library.

It seems to me the outlier is the ```Storage Data``` cmdlets, which shall be in a specialized namespace, not the ```Storage Management``` ones.

@markcowl Thoughts?